samples/line-horizontal.html

```diff
@@ -86,30 +87,35 @@
 					text: 'Line chart (hotizontal scroll) sample'
 				},
 				scales: {
-					xAxes: [{
+					'x': {
 						type: 'realtime',
 						realtime: {
 							duration: 20000,
-							refresh: 1000,
+							// refresh: 1000,
 							delay: 2000,
 							onRefresh: onRefresh
 						}
-					}],
-					yAxes: [{
-						scaleLabel: {
-							display: true,
-							labelString: 'value'
+					},
+					'y': {
+						'type': 'linear',
+						'scaleLabel': {
+							'display': true,
+							'labelString': 'value'
+						},
+						'ticks': {
+							'min': 0,
+							'suggestedMax': 10
 						}
-					}]
+					}
 				},
-				tooltips: {
-					mode: 'nearest',
-					intersect: false
-				},
-				hover: {
-					mode: 'nearest',
-					intersect: false
-				}
+				// tooltips: {
```

Comment:
we shouldn't have commented out code. we should fix it, remove it if no longer applicable, or at least add a comment explaining why it's commented out and what would need to be done to address it